src/libraries/System.Text.Json/tests/Common/NumberHandlingTests.cs

```diff
@@ -1057,6 +1057,27 @@ private async Task AssertUnsupportedFormatThrows<T>()
             }
         }
 
+#if NETCOREAPP
+        [Fact]
+        public async Task InvalidNumberFormatThrows()
+        {
+            await AssertInvalidNumberFormatThrows<Int128>("170141183460469231731687303715884105728"); // MaxValue + 1
```

Comment:
Nit: could this be a theory?

Reply:
Yes it could, but this is pattern used by the nearby tests in this file.